handler.js

```diff
@@ -1,33 +1,15 @@
 'use strict';
-const request = require('request');
-const helpers = require('./helpers/helperFunctions');
-const validations = require('./helpers/validations');
-const Json2csvParser = require('json2csv').Parser;
-const config = require('./config/foursquare');
 
+const helpers = require('./helpers/helperFunctions');
 
 module.exports.getVenues = (event, context, callback) => {
 
 try{
- const inputs = inputsData(JSON.parse(event.body));
-
-
-foursquareApiData(inputs).then(function(data){
+ const inputs = helpers.processInputsData(JSON.parse(event.body));
+ helpers.getFoursquareApiData(inputs).then(function(data){
 
-const columnNames = ["Name", "City", "Street", "Latitude", "Longitude"];
+const response = helpers.responseCsvVenue(data);
```

Comment:
Better name for this function will be something like `getResponseCSVVenue`, name should contain an action